contrib/gitian-descriptors/gitian-osx.yml

```diff
@@ -132,6 +133,31 @@ script: |
 
     make osx_volname
     make deploydir
+
+    # Hack to work around differences in the behavior of native_cctools' codesign_allocate and Apple distributed codesign and codesign_allocate.
+    # Specifically, the signature is created by appending it to the end of the binary and then updating the size for the section __LINKEDIT
+    # The "vmsize" attribute of this section is rounded to 0x1000, 0x2000, or 0x4000 depending on the architecture. For some reason, this also happens
+    # to differ between what we can build ourselves in native_cctools (which uses Apple's published source code) and what Apple distributes
+    # as "Xcode command line tools". It appears that the Apple distributed tool rounds to 0x2000.
+    #
+    # Fortunately codesign_allocate will not reduce the "vmsize" attribute, and when the signature is appended, the difference in actual size and
+    # "vmsize" will not be present in the final binary.
+    #
+    # Thus we can workaround this by calculating an overestimate of what the signature will be and then using codesign_allocate to preallocate
+    # space for the code signature. Because the same codesign_allocate is used in this step, the rounding problem will not be present.
+    # Since the space allocated for the signature should be way larger than actually needed, the signing step will not reduce "vmsize" so we
```

Comment:
This line seems outdated now.

Reply:
Which part?

Reply:
It's not way larger overestimate anymore.

Reply:
Fixed